src/Chapter3.hs

```diff
@@ -460,6 +490,20 @@ Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
 
+data MealType
+  = Suhur
+  | Breakfast
+  | Tiffin
+  | Brunch
+  | Elevenses
+  | Lunch
+  | AfternoonTea
+  | HighTea
+  | Dinner
+  | Supper
+  | SiuYeh
```

Comment:
Very nice food selection! :heart_eyes: 
Btw, what is `Suhur` and `SiuYeh`?

Reply:
Did some reseach about meals and found https://en.wikipedia.org/wiki/Siu_yeh and https://en.wikipedia.org/wiki/Suhur